tests/checks/mock/test_spark.py

```diff
@@ -206,6 +212,69 @@ def raise_for_status(self):
             body = f.read()
             return MockStandaloneResponse(body, 200)
 
+def standalone_requests_pre20_get_mock(*args, **kwargs):
```

Comment:
Ideally we could maybe refactor some code out of `standalone_requests_pre20_get_mock()` and `standalone_requests_get_mock()` so we'd have less duplicated code?

maybe something like:

```
class MockStandaloneResponse:
    text = ''
 
    def __init__(self, json_data, status_code):
         self.json_data = json_data
         self.status_code = status_code
         self.text = json_data

    def json(self):
        return json.loads(self.json_data)

    def raise_for_status(self):
        return True

def standalone_request_get_mock(*args, **kwargs):
    if args[0] == STANDALONE_APP_URL:
        with open(Fixtures.file('spark_standalone_apps'), 'r') as f:
        body = f.read()
        return MockStandaloneResponse(body, 200)
    # REMAINING logic

def standalone_request_pre20_get_mock(*args, **kwargs):
    if args[0] == STANDALONE_SPARK_APP_URL:
        with open(Fixtures.file('spark_apps_pre20'), 'r') as f:
            body = f.read()
            return MockStandaloneResponse(body, 200)
    else:
        return standalone_request_get_mock(args, kwargs)
```